gems/pending/appliance_console/utilities.rb

```diff
@@ -21,26 +24,21 @@ def self.bail_if_db_connections(message)
     end
 
     def self.db_host_type_database
-      require 'open4'
-      out = nil
-      err = nil
-      Open4::popen4("cd #{RAILS_ROOT} && script/rails runner 'puts MiqDbConfig.current.options[:host]|| \"localhost\"; puts MiqDbConfig.current.options[:name]; puts MiqDbConfig.current.options[:database]'") do |pid, stdin, stdout, stderr|
-        out = stdout.read
-        err = stderr.read
-      end
-      out = out.chomp.split("\n")
-      host, type, database = out[-3..-1]
-      unless [host, type, database].all? { |res| res.kind_of?(String) }
-        File.open(LOGFILE, 'a') do |f|
-          f.puts "db_host_type_database: Failed to detect some/all DB configuration"
-          f.puts "Output: #{out.inspect}" if out.to_s.length > 0
-          f.puts "Error:  #{err.inspect}" if err.to_s.length > 0
-        end
+      result = AwesomeSpawn.run("bin/rails runner",
```

Comment:
lets not over think this change too much.
This causes the multi second lag while going into appliance console.

Feel it would be better to just read in `database.yml`. In the end, that is basically what happens in the rails console anyway.